packages/@uppy/companion/src/server/socket.js

```diff
@@ -48,6 +48,16 @@ module.exports = (server) => {
     emitter().emit(`connection:${token}`)
     emitter().on(token, send)
 
+    ws.on('error', (err) => {
+      // https://github.com/websockets/ws/issues/1543
+      // https://github.com/websockets/ws/blob/b73b11828d166e9692a9bffe9c01a7e93bab04a8/test/receiver.test.js#L936
+      if (err instanceof RangeError && 'code' in err && err.code === 'WS_ERR_UNSUPPORTED_MESSAGE_LENGTH') {
+        logger.warn('WebSocket message too large', 'websocket.error', shortenToken(token))
```

Comment:
I think it should be an error, ideally we would still want a pager 
```suggestion
        logger.error('WebSocket message too large', 'websocket.error', shortenToken(token))
```